core/celo_genesis.go

```diff
@@ -150,11 +138,56 @@ func celoGenesisAccounts(fundedAddr common.Address) GenesisAlloc {
 				common.HexToHash("0x2"): devBalance32, // _totalSupply
 			},
 		},
+		mockOracleAddr: {
+			Code:    mockOracleBytecode,
+			Balance: big.NewInt(0),
+			Storage: map[common.Hash]common.Hash{
+				common.HexToHash("0x0"): common.BigToHash(rateNumerator),
+				common.HexToHash("0x1"): common.BigToHash(rateDenominator),
+			},
+		},
+		mockOracleAddr2: {
+			Code:    mockOracleBytecode,
+			Balance: big.NewInt(0),
+			Storage: map[common.Hash]common.Hash{
+				common.HexToHash("0x0"): common.BigToHash(rateNumerator2),
+				common.HexToHash("0x1"): common.BigToHash(rateDenominator),
+			},
+		},
 		DevAddr: {
 			Balance: DevBalance,
 		},
 		FaucetAddr: {
 			Balance: faucetBalance,
 		},
 	}
+
+	// FeeCurrencyDirectory
+	devAddrOffet1 := common.Hash{}
+	copy(devAddrOffet1[11:], DevAddr.Bytes())
+	feeCurrencyDirectoryStorage := map[common.Hash]common.Hash{
+		// owner, slot 0 offset 1
+		common.HexToHash("0x0"): devAddrOffet1,
```

Comment:
`devAddrOffset1` perhaps?
```suggestion
	devAddrOffset1 := common.Hash{}
	copy(devAddrOffset1[11:], DevAddr.Bytes())
	feeCurrencyDirectoryStorage := map[common.Hash]common.Hash{
		// owner, slot 0 offset 1
		common.HexToHash("0x0"): devAddrOffset1,
```